packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/expiration-modal.tsx

```diff
@@ -2,44 +2,33 @@ import React from 'react';
 import { Button, Div100vhContainer, Modal, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import Expiration from './expiration';
-import { observer } from '@deriv/stores';
 
 type TMultipliersExpirationModalProps = {
     is_open: boolean;
     toggleModal: () => void;
 };
 
-const MultipliersExpirationModal = observer(({ is_open, toggleModal }: TMultipliersExpirationModalProps) => (
+const MultipliersExpirationModal = ({ is_open, toggleModal }: TMultipliersExpirationModalProps) => (
     <Modal
         is_open={is_open}
         toggleModal={toggleModal}
         has_close_icon={false}
         should_header_stick_body={false}
-        height='auto'
-        width='calc(100vw - 3.2rem)'
```

Comment:
`<Modal />` applies` .dc-modal__container` className, which already contains same `width` and `height`. No need in duplicating